src/API/Google/AdsAssetGroup.php

```diff
@@ -362,4 +405,50 @@ protected function convert_asset_group( GoogleAdsRow $row ): array {
 			'display_url_path' => [ $row->getAssetGroup()->getPath1(), $row->getAssetGroup()->getPath2() ],
 		];
 	}
+
+	/**
+	 * Send a batch of operations to mutate an asset group.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param MutateOperation[] $operations
+	 *
+	 * @return int If the asset group operation is present, it will return the asset group id otherwise 0 for other operations.
+	 * @throws ApiException If any of the operations fail.
+	 */
+	protected function mutate( array $operations ): int {
+		$responses = $this->client->getGoogleAdsServiceClient()->mutate(
+			$this->options->get_ads_id(),
+			$operations
+		);
+
+		foreach ( $responses->getMutateOperationResponses() as $response ) {
+			if ( 'asset_group_result' === $response->getResponse() ) {
+				$asset_group_result = $response->getAssetGroupResult();
+				return $this->parse_asset_group_id( $asset_group_result->getResourceName() );
```

Comment:
💅  This could throw an Exception but only ApiException is being reflected in the PHPDOCS